test/unit/parser.test.js

```diff
@@ -4,6 +4,7 @@ const MarkBind = require('../../src/lib/markbind/src/parser.js');
 const { USER_VARIABLES_DEFAULT } = require('./utils/data');
 
 jest.mock('fs');
+jest.mock('../../src/util/logger');
```

Comment:
👀

Reply:
This was needed back when the markdown-it plugin logged errors. The current plugin fails silently and prints "Invalid Date". I'll remove it since it is no longer needed